Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -68,6 +68,12 @@ final class PageActionMenu: UIViewController, UIGestureRecognizerDelegate, Theme
         checkSwipeDown()
         guard traitCollection.userInterfaceIdiom == .pad else { return }
         contentSizeObserver = tableView.observe(\.contentSize) { [weak self] tableView, _ in
+            // Ecosia: Update height for iPad
+            // self?.preferredContentSize = CGSize(width: 350, height: tableView.contentSize.height)
```

Comment:
This line still exists below the introduced code, is that expected? If so, would just say we need to remove this commented line not to be duplicated.

Reply:
The reason I made it so as I foresaw a future conflict when upgrading. By commenting it that way we'll have a better separated batch of code changes and would be easier to perform a choose left/right/both when solving conflicts.